rust/blockstore/src/arrow/block/value/u32_value.rs

```diff
@@ -38,8 +40,8 @@ impl ArrowWriteableValue for u32 {
         }
     }
 
-    fn get_delta_builder() -> BlockStorage {
-        BlockStorage::UInt32(SingleColumnStorage::new())
+    fn get_delta_builder(mutation_ordering_hint: BuilderMutationOrderHint) -> BlockStorage {
+        BlockStorage::UInt32(SingleColumnStorage::new(mutation_ordering_hint))
     }
```

Comment:
we should be able to eventually remove this if we wish and simply have a `type DeltaBuilder = SingleColumnStorage<u32>` GAT after Hammad's changes